src/execution_plan/ops/op.h

```diff
@@ -63,16 +63,19 @@ struct OpBase {
     OPType type;                // Type of operation
     fpInit init;                // Called once before execution.
     fpConsume consume;          // Produce next record.
+    fpConsume profile;          // Profiled version of consume.
     fpReset reset;              // Reset operation state.
     fpFree free;                // Free operation.
     char *name;                 // Operation name.
     Vector *modifies;           // List of aliases, this op modifies.
     struct OpBase **children;   // Child operations.
     int childCount;             // Number of children.
+    int profileRecordCount;     // Number of records generated.
```

Comment:
I don't really feel like this belongs here, but maybe we'll be able to come up with a cleaner solution once we've started writing profiler routines.

Reply:
I'll try to come up with a different design.